templates/ui-service.yaml

```
@@ -25,3 +28,5 @@ spec:
       targetPort: 8200
   type: {{ .Values.ui.serviceType | default "ClusterIP" }}
 {{- end -}}
+
+{{ end }}
```

New line at the end of file 
```suggestion
{{ end }}

```